src/views/dataset/MultiSourceConfiguration.vue

```diff
@@ -102,29 +88,36 @@ export default class NewDataset extends Vue {
     return this.$route.params.id;
   }
 
+  // Create a short string describing the array with some elements from the array
+  // For example: ["foo", "bar", "foobar", "barfoo"] => "foo, bar, foobar..."
+  arrayToExampleValues(arr: string[]) {
+    let values = arr.slice(0, 3).join(", ");
+    if (arr.length > 3) {
+      values = String.prototype.concat(values, "...");
+    }
+    return values;
```

Comment:
I think it won't work if your items are VERY long.

Your limit could be instead a number of characters:
``` 
function truncate(str, n){
  return (str.length > n) ? str.slice(0, n-1) + '&hellip;' : str;
};
```

Or even better, [handle it at CSS level](https://stackoverflow.com/a/9792134).

Reply:
I prefer doing it at JS level to avoid having a big chunk of text almost unused in the DOM.

Reply:
By doing it as JS level, you are not very flexible when it comes to dynamic layout. What if user zooms the ui (CTRL+'+') or if the UI is visible from a mobile device. If tomorrow you change the "language" or the font size, you will also have to modify the limit in the number of characters.